docs/tools/asf_tools_api.md

```diff
@@ -1,3 +1,5 @@
 # `asf_tools` *v0.4.0* API Reference
 
-::: asf_tools
+{{ get_content('https://raw.githubusercontent.com/ASFHyP3/asf-tools/v0.4.0/asf_tools/README.md') }}
```

Comment:
While this pin-points the issue to *something* going on with the `mkdocstrings` plugin, this change make it so our API reference:
https://hyp3-docs.asf.alaska.edu/tools/asf_tools_api/

renders as just our `README.md`:
https://hyp3-docs.asf.alaska.edu/tools/asf_tools/

IF we really can't get `mkdocstrings` to work, which is highly undesirable, it'd be better to eliminate this API reference page entirely.